types/stylelint/index.d.ts

```diff
@@ -329,6 +329,7 @@ declare namespace stylelint {
 		readonly 'no-irregular-whitespace': Promise<Rule>;
 		readonly 'no-unknown-animations': Promise<Rule>;
 		readonly 'no-unknown-custom-properties': Promise<Rule>;
+		readonly 'no-unknown-custom-media': Promise<Rule>;
```

Comment:
[suggest] Let's reorder alphabetically:

```suggestion
		readonly 'no-unknown-custom-media': Promise<Rule>;
		readonly 'no-unknown-custom-properties': Promise<Rule>;
```